dev/ci/ci-common.sh

```diff
@@ -46,6 +46,40 @@ done
 
 mathcomp_CI_DIR="${CI_BUILD_DIR}/math-comp"
 
+# [git_download project dest] will download the given [ref] of [project]
+# and unpack it in [dest] if the folder does not exist already;
+# if it does, it will do nothing except print a warning
+# (this can be useful when testing make ci-* locally).
+# When the variable $CI is undefined, it uses git clone to perform the
+# download.
+git_download()
+{
+  local PROJECT=$1
+  local DEST=$2
+
+  if [ -d "$DEST" ]; then
```

Comment:
Oh I see there's this bit

Reply:
This was inspired by your first review comment about `mkdir -p` succeeding too often.